include/ck_tile/ops/gemm/block/block_gemm_asmem_bsmem_creg_v1_default_policy.hpp

```diff
@@ -1,10 +1,11 @@
 // SPDX-License-Identifier: MIT
-// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
we can remove the dates from the copyright boilerplate

Reply:
Let's leave it to bulk codemod rewriting the headers when it's necessary

Reply:
Actually, some practice just updates the year whenever the code is updated. https://opensource.stackexchange.com/a/6390

The logic in `script/check_copyright_year.sh` follows this logic as a part of pre-commit. (as pre-commit scripts only run for changed files).